docs/README_Docker_and_RA.md

```diff
@@ -310,14 +312,17 @@ docker image ls
 
 You should see at least two images, `us-docker.pkg.dev/evmchain/raiko` and `us-docker.pkg.dev/evmchain/pccs`.
 
-7. Create a `.env` using the `.env.sample` template. Ensure `SGXGETH=true` is in the `.env`.
+7. Create a `.env` using the `.env.sample` template. 
 
 You can copy the template with the following:
 
 ```
 cp .env.sample .env
 ```
 
+If you are running Raiko for Taiko Hekla, ensure `SGXGETH=true`, `NETWORK=taiko_a7` and `L1_NETWORK=holesky` in `.env`.
+If you are running Raiko for Taiko Alethia, ensure `SGXGETH=false`, `NETWORK=taiko_mainnet` and `L1_NETWORK=ethereum` in `.env`.
```

Comment:
delete the SGXGETH in alethia, it's now a exist-or-not flag, not a true/false checking..